Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -419,6 +419,17 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters,
     // Reply immediately so that the identity is available as soon as possible.
     completionHandler(ProcessIdentity { ProcessIdentity::CurrentProcess });
 
+    m_isLockdownModeEnabled = parameters.isLockdownModeEnabled;
```

Comment:
We're getting this flag from 2 places, once early on via XPC and then once again via this WebProcessCreationParameters struct. Why do we need it in 2 places?